Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "ClimaSeaIce"
 uuid = "6ba0ff68-24e6-4315-936c-2e99227c95a4"
 authors = ["Climate Modeling Alliance and contributors"]
-version = "0.3.5"
+version = "0.3.6"
```

Comment:
@simone-silvestri, was there a breaking change? should we go 0.4.0?

Reply:
not really, just the internals changed, user interface and computations should stay exactly the same

Reply:
Okie!